src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1394,7 +1394,6 @@ class S6Importer final : public IParkImporter
             else
```

Comment:
But what happens when line 1391 is nullptr. There is no SetTrackType now for it. Thats what I was trying to say before but didn't account for L1374.

Reply:
I've changed this now, it should no longer have a floating TrackType